opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -2522,6 +2522,19 @@ namespace Opm {
         }
     }
 
+    template <typename TypeTag>
+    void
+    BlackoilWellModel<TypeTag>::
+    assignNodeValues(const int                              reportStepIdx,
+                     const Schedule&                        sched,
+                     std::map<std::string, data::NodeData>& nodevalues) const
+    {
+        nodevalues.clear();
+        for (const auto& np : node_pressures_) {
+            nodevalues.emplace(np.first, data::NodeData{np.second});
+        }
```

Comment:
Could alternative write this using structured bindings to avoid having to refer to `.first` and `.second`.
```C++
for (const auto& [node, pressure] : node_pressures_) {
    nodeValues.emplace(node, data::nodeData{ pressure });
}
```

Reply:
I am a bit on the fence there. The code in the loop is of course nicer, but the for loop statement looks a bit messier (and the feature is maybe still somewhat exotic). My very basic rule of thumb, to make a variable if I use an expression twice, would seem to say don't bother. However I need to get the structured bindings under my skin, so I'll change...